packages/devtools/src/commands/isElementDisplayed.js

```diff
@@ -0,0 +1,9 @@
+export default function isElementDisplayed ({ elementId }) {
```

Comment:
let me change the behavior in WebdriverIO since Puppeteer can actually determine visibility

Reply:
so we will call `isElementDisplayed` in the `isVisibleWithinViewport` command of WebdriverIO so that the command is not unused.

Reply:
Can't understand fully, why don't you just change a bit condition in isDisplayed.js to call protocol command instead of using script?
Am I understanding something wrong?

Reply:
The puppeteer isVisible protocol command can just check if the element is within the viewport: https://pptr.dev/#?product=Puppeteer&version=v1.19.0&show=api-elementhandleisintersectingviewport

Reply:
I don't mind, but it won't be used neither if we don't change condition in webdriverio isDisplayed.js

Reply:
I don't mind, but it won't be used neither if we don't change condition in webdriverio isDisplayed.js

Reply:
@mgrybyk I added a condition is WebdriverIOs `isVisibleWithinViewport` command